apps/web/src/features/calendar/calendar.css

```
@@ -194,12 +211,105 @@
+	.calendar-event-content-layout-single-line
+	.calendar-event-location {
+	display: none;
+}
+
+.calendar-view .fc .fc-event:hover {
+	z-index: 4;
+}
+
+.calendar-view .fc .fc-event:focus-visible {
+	z-index: 4;
+	box-shadow: 0 0 0 1px currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)